apidoc/Titanium/UI/TextArea.yml

```diff
@@ -366,6 +366,14 @@ properties:
     default: <Titanium.UI.KEYBOARD_DEFAULT>
     platforms: [android, iphone, ipad]
 
+  - name: lines
+    summary: Line count of text field input.
+    description: Sets the initial size of lines that the field will have at start.
```

Comment:
Let's make it clear that `lines` sets the TextArea height by line height... and get rid of the "text field" references since this only applies to text areas.

How about the following...
```
summary: Number of lines tall the text area height will be, if set.
description: Sets the height of the text are by line height. Set to -1 to respect the view's height property instead.
```